### PR TITLE
Fix micro-account scalping sizing floor

### PR DESCRIPTION
**Pull request by irfndi**
## Summary
- extract executable futures sizing constraints into `internal/app/autonomy`
- floor effective micro-account capital sizing to the exchange minimum notional when policy caps are too small
- teach AI scalping prompts and execution guards that `size_pct` is direct notional share of wallet and must stay inside the executable band

## Verification
- `go test ./internal/app/autonomy -run 'TestEvaluateScalpingPolicy|TestResolveExecutableSizingConstraints'`
- `go test ./internal/services -run 'TestAIScalpingService_(BuildUserPrompt_UsesEffectiveThresholdsOnly|DynamicRiskThresholds_FloorsToExecutableMinimum|ExecuteDecision_BumpsSizeToExecutableMinimum|DynamicRiskThresholds_RecoveryModes)'`
- `make build`
- upgraded local install with `./install.sh --enable-autostart`
- live log verification on 2026-03-09 23:56:59 WIB showed `min_confidence=0.65` and `max_capital_pct=12.78` for the 46.93 USDT Bitget wallet after restart

## Follow-up
- `NeuraTrade-azj` Harden scalping structured-output path
- `NeuraTrade-v09` Refactor autonomy logic out of internal/services

## Summary by Sourcery

Enforce exchange-level executable sizing constraints for AI scalping, ensuring micro-account position sizes respect Bitget futures minimums and are reflected consistently across policy evaluation, prompts, and execution.

New Features:
- Introduce reusable executable sizing constraint resolution in the autonomy layer, including Bitget futures minimum notional and derived wallet percentage thresholds.
- Expose executable sizing fields on the trading portfolio and surface an executable size band and sizing semantics in AI scalping prompts.

Bug Fixes:
- Prevent AI scalping on micro accounts from choosing position sizes below the exchange minimum by flooring effective max capital and executed sizes to the executable minimum when necessary.

Enhancements:
- Wire executable sizing constraints into scalping policy evaluation and dynamic risk thresholds so policy caps cannot fall below the exchange-mandated minimum size.
- Adjust deterministic fallback sizing and order execution guards to honor the executable size floor while keeping sizes within configured caps.
- Clarify AI scalping trading rules so size_pct is treated as a direct notional share of wallet independent of leverage.

Tests:
- Add unit tests for executable sizing constraint resolution, policy flooring to the exchange minimum, dynamic risk threshold behavior, prompt construction with sizing context, and execution behavior that bumps too-small sizes up to the executable floor.

## Summary by CodeRabbit

* **New Features**
  * Executable sizing surfaced in prompts, portfolio metadata, and enforced across the trading flow (executable size band, estimated notional/initial margin).
  * Operational mode sync (live/dry/paper) integrated into scalping execution.
  * New env var NEURATRADE_BITGET_FUTURES_MIN_NOTIONAL_USDT to override Bitget futures minimum notional.

* **Bug Fixes / Behavior**
  * Orders and sizing honor exchange minimums and wallet feasibility; decisions are adjusted or held when non-executable.
  * Micro-account no-fill recovery and dynamic risk thresholds respect executable floors.
  * Realized PnL scoped queries preferred when available.

* **Tests**
  * Expanded coverage for sizing, policy floors, no-fill recovery, mode sync, performance stats, and order executor fallbacks.

* **Documentation / Chores**
  * Updated workflow docs and .gitignore.

---

<a href="https://app.devin.ai/review/irfndi/neuratrade/pull/241" target="_blank">
  <picture>
    <source media="(prefers-color-scheme: dark)" srcset="https://static.devin.ai/assets/gh-open-in-devin-review-dark.svg?v=1">
    Open with Devin
  </picture>
</a>

### CODE REVIEW

File: services/backend-api/internal/app/autonomy/scalping_cycle.go [hunk 1]
```diff
@@ -292,6 +293,12 @@ func EvaluateScalpingPolicy(input ScalpingCycleInput, cfg ScalpingPolicyConfig)
 		}
 		policy.MaxConcurrentPositions = cfg.MicroMaxConcurrent
 	}
+	if input.ExecutionMinCapitalPct > 0 &&
+		input.ExecutionMinCapitalPct <= 100 &&
+		input.ExecutionMinCapitalPct > policy.EffectiveMaxCapitalPct {
+		policy.EffectiveMaxCapitalPct = input.ExecutionMinCapitalPct
+		policy.PolicyAdjustments = append(policy.PolicyAdjustments, "exchange_min_notional_floor")
+	}
```
[the remark this answers is not in the document]
Comment (irfndi) [reply services/backend-api/internal/app/autonomy/scalping_cycle.go]: Addressed in 2a1e1963. `ExecutionMinCapitalPct` is now a `decimal.Decimal`, and `EvaluateScalpingPolicy` explicitly marks `> 100` as `exchange_min_notional_block` with an empty executable band (`EffectiveMaxCapitalPct = 0`). I added a focused policy test for that branch.
